### ISSUE TITLE
[Enhancement] EntityTopicOperator and EntityUserOperator should not use hard coded values for health checks

### ISSUE THREAD

**Issue by btison** [strimzi/strimzi-kafka-operator#1616]
**Is your feature request related to a problem? Please describe.**
EntityTopicOperator and EntityUserOperator should not use hard-coded settings for health check initial delay and timeout

**Describe the solution you'd like**
Health check initial delay and timeout configurable in the kafka CR
```
  spec:
    entityOperator:
      tlsSidecar:
        livenessProbe:
          initialDelaySeconds: 20
          timeoutSeconds: 5
        readinessProbe:
          initialDelaySeconds: 15
          timeoutSeconds: 5
        resources:
          limits:
            cpu: 50m
            memory: 100M
          requests:
            cpu: 50m
            memory: 100M
      topicOperator:
        livenessProbe:
          initialDelaySeconds: 20
          timeoutSeconds: 5
        readinessProbe:
          initialDelaySeconds: 20
          timeoutSeconds: 5
        resources:
          limits:
            cpu: 250m
            memory: 500M
          requests:
            cpu: 150m
            memory: 250M
      userOperator:
        livenessProbe:
          initialDelaySeconds: 20
          timeoutSeconds: 5
        readinessProbe:
          initialDelaySeconds: 20
          timeoutSeconds: 5
        resources:
          limits:
            cpu: 250m
            memory: 250M
          requests:
            cpu: 150m
            memory: 150M
```

**Comment by scholzj** [comment]
This has been addressed by the PR #1619 and can be now closed.

**Comment by scholzj** [comment]
@btison Sorry, I wanted to close it when I wrote the comment that it can be closed. I guess I pressed the wrong button I guess :-o.
